style.css

```diff
@@ -333,3 +345,256 @@ about me section
+  margin-right: 24px;
+}
+
+.contactme h2 {
+  margin-top: 60px;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 700;
+  color: #172b4d;
+  margin-bottom: 58px;
+}
+
+.contactme form {
+  display: flex;
+  flex-direction: column;
+}
+
+/* add styling to form elements -ram */
+.form-element {
+  height: 48px;
+  border: 2px solid #d0d9d4;
+  color: #6b778c;
+  border-radius: 4px;
+  margin-bottom: 24px;
+  font-size: 15px;
+  line-height: 24px;
+  text-align: left;
+  padding: 15px;
+}
+
+.msg-text {
+  height: 30%;
+}
+
+.form-button {
+  margin: 0 auto;
+}
+
+.social-footer {
+  margin-top: 10vh;
+  display: flex;
+  flex-direction: column;
+  gap: 3vh;
+  align-items: center;
+}
+
+.postal-email-id {
+  font-family: "Inter", sans-serif;
+  font-size: 2.5rem;
+  margin-bottom: 189px;
+}
+
+.social-footer hr {
+  margin-top: 134px;
+  margin-bottom: 20px;
+  border-top: 4px solid black;
+  width: 30%;
+  display: inline-flex;
+  align-items: center;
+}
+
+.firstlastname {
+  display: none;
+}
+
+.dividerfoo {
+  display: none;
+}
+
+@media screen and (min-width: 800px) {
+  .bigheader {
+    background-image: url(./images/big-hero.png);
+    height: 950px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: relative;
+  }
+
+  header {
+    background: transparent;
+    flex-direction: column-reverse;
+    justify-content: center;
+  }
+
+  header h5,
+  header img {
+    display: none;
+  }
+
+  .topnav {
+    display: inline-block;
+  }
+
+  .topnav a {
+    text-decoration: none;
+    color: #344563;
+    font-size: 15px;
+    line-height: 20px;
+    padding-right: 32px;
+  }
+
+  .headline {
+    margin: 0;
+    padding: 0;
+    background-image: none;
+    display: block;
+  }
+
+  h1 {
+    font-size: 48px;
+    line-height: 60px;
+    margin-top: 160px;
+    font-weight: 800;
+    margin-bottom: 24px;
+  }
+
+  .headline p {
+    line-height: 32px;
+    width: 920px;
+    padding: 0 80px;
+    margin-top: 0;
+    margin-bottom: 24px;
+  }
+
+  .icons-contact {
+    margin-bottom: 74px;
+    height: 40px;
+  }
+
+  .icons-contact img {
+    padding-right: 20px;
+  }
+
+  .down {
+    margin-right: 40px;
+  }
+
+  .works {
+    margin-bottom: 140px;
+    margin-top: 102px;
+  }
+
+  .works hr {
+    display: none;
+  }
+
+  .work_cards {
+    grid-template-columns: auto auto auto;
+  }
+
+  .card {
+    width: 369px;
+  }
+
+  .about_desc {
+    margin: 52px 260px;
+  }
+
+  .about_desc h2 {
+    margin-bottom: 0;
+  }
+
+  .about_desc p {
+    padding: 24px 60px;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .about_desc button {
+    margin-top: 0;
+    margin-bottom: 0;
+    width: 135px;
+  }
+
+  .card_about {
+    grid-template-columns: auto auto auto;
+    margin-top: 0;
+    gap: 24px;
+  }
+
+  .cardabt {
+    width: 369px;
+  }
+
+  .contactme {
+    margin: 64px 135px 0 135px;
+    display: grid;
+    grid-template-columns: auto auto;
+  }
```

Comment:
- [ ] Kindly implement the `background image` for `contact me` section also for desktop version following the given [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=qfJVZ1dGJDngRA30-0), so that it looks more professional and eye catchy.

- ***In the picture top one is your design, and below one is given figma design.***

![Screenshot from 2022-12-08 19-21-54](https://user-images.githubusercontent.com/60083437/206457941-4fa311fe-b318-4cd6-b268-c111ffd4feba.png)